src/documentation/runners/mapreduce.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: "Apache Hadoop MapReduce Runner"
+permalink: /documentation/runners/mapreduce/
```

Comment:
I think since we will not link from the overview and will not link from the header, you should follow this up by adding a link from the work-in-progress page.

Reply:
done